hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/PersistentState.java

```diff
@@ -33,15 +34,17 @@
 
 @Singleton
 public class PersistentState {
-  public static final Log log = LogFactory.getLog(PersistentState.class);
+  private final Log log = LogFactory.getLog(PersistentState.class);
+
   private static final String FRAMEWORK_ID_KEY = "frameworkId";
   private static final String NAMENODES_KEY = "nameNodes";
   private static final String JOURNALNODES_KEY = "journalNodes";
   private static final String DATANODES_KEY = "dataNodes";
   private static final String NAMENODE_TASKNAMES_KEY = "nameNodeTaskNames";
   private static final String JOURNALNODE_TASKNAMES_KEY = "journalNodeTaskNames";
-  private ZooKeeperState zkState;
-  private SchedulerConf conf;
+
+  private State zkState;
```

Comment:
Let's keep naming consistent. This should be hdfsFrameworkConfig everywhere our frameworkConfig everywhere. We need to be as consistent as possible.